src/components/ListTable/index.vue

```diff
@@ -71,7 +71,11 @@ export default {
       }
       const defaults = {}
       for (const [k, v] of Object.entries(actions)) {
-        defaults[k] = this.hasActionPerm(v.action)
+        if (k === 'hasImport') {
+          defaults[k] = this.hasActionPerm(v.action) || this.hasActionPerm('change')
+        } else {
+          defaults[k] = this.hasActionPerm(v.action)
+        }
```

Comment:
for (const [k, v] of Object.entries(actions)) {
        let hasPerm = v.action.split('|').reduce(
          (previousValue, currentValue) => previousValue || this.hasActionPerm(currentValue),
          false
        )
        if (v.checkRoot) {
          hasPerm = hasPerm && !this.currentOrgIsRoot
        }
        defaults[k] = hasPerm
        console.log('default: ', defaults)
      }